assets/scss/custom/_styles.scss

```diff
@@ -240,6 +240,29 @@ body {
   padding: 0.25rem 0.6rem;
 }
 
+//Ratings table
+.table-ratings {
+  table-layout: fixed;
+  font-size: 0.9rem;
+}
+
+//Map column for Ratings table
+.table-ratings tr th:nth-child(1) {
+  width: 200px;
```

Comment:
not a fan of putting dimensions in pixels, since they don't adapt with font size. So if the user changes its font size, it gets out of whack easily. The default font size is 16px, that means 200px is `12.5rem`.

Reply:
Have updated to use REM and confirmed it looks the same.